tests/Doctrine/Tests/ORM/Tools/Console/Command/SchemaTool/CreateCommandTest.php

```diff
@@ -12,6 +12,9 @@ public function testItPrintsTheSql(): void
     {
         $tester = $this->getCommandTester(CreateCommand::class);
         $tester->execute(['--dump-sql' => true]);
-        self::assertStringContainsString('CREATE TABLE keyboard', $tester->getDisplay());
+
+        $statements = $tester->getDisplay();
+        self::$sharedConn->executeStatement($statements);
+        self::assertTrue(true);
```

Comment:
Please use `@doesNotPerformAssertions` instead, here and below, it's better than asserting it returns an integer as I previously suggested, and is certainly better than asserting that true is true.